dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -366,6 +354,7 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
 
                 GetWaitHandleForArgs().create();
 
+                s_taskClsid = taskClsid;
```

Comment:
This should be in a lock

Reply:
Also UnregisterActivator should remove the long running sink from LRP for Packaged apps

Reply:
fix unregister too to remove foreground sink with LRP for packaged apps